docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,181 @@
+* Randomly generated floats = ``[[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1], [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]]``
+* Assigned classes = ``[[1, 1, 2, 2, 2, 2, 2, 2, 2, 2], [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]]``
+
+Example 3 - 1D tensor, without replacement
+* Let *input* = ``[0.1, 0.5, 0.4]``, *num_samples* = 2, *log_probs* = false, *replacement* = false
+* CDF of *input* = ``[0.1, 0.6, 1]``
+* Randomly generated floats = ``[0.3, 0.6]``
+* In a loop:
+* For a value of 0.3, a class with idx *1* is selected 
+* Therefore, in CDF, for every class starting with idx *1* subtract the probability of class *1* = 0.5
+* CDF = ``[0.1, 0.6 - 0.5, 1.0 - 0.5]`` = ``[0.1, 0.1, 0.5]``
+* Normalize CDF by dividing by last value: CDF = ``[0.2, 0.2, 1.0]``
+* Take the next randomly generated float, here 0.6, and repeat until all random samples have assigned classes
+* Assigned classes = ``[1, 2]``
+
+
+**Attributes**:
+
+* ``output_type``
+
+  * **Description**: the type of the output. Determines generation algorithm and affects resulting values. Output numbers generated for different values of *output_type* may not be equal.
+  * **Range of values**: "i4", "i8", "i16", "i32", "i64", "f32", "u4", "u8", "u16", u32", "u64".
+  * **Type**: string
+  * **Required**: *Yes*
+
+* ``with_replacement``
+
+  * **Description**: controls whether to sample with replacement (classes can be sampled multiple times).
+  * **Range of values**: `true`, `false`
+      * ``true`` - class indices can be sampled multiple times.
+      * ``false`` - class indices will not repeat in the output and the size of *input*'s ``class_size`` dimension is required to be larger or equal to *num_samples* value. Might affect performance.
+  * **Type**: `bool`
+  * **Required**: *Yes*
+
+* ``log_probs``
+
+  * **Description**: allows to control whether *inputs* should be treated as probabilities or unnormalized log probabilities.
+  * **Range of values**: `true`, `false`
+      * ``true`` - set values in *inputs* are unnormalized log probabilities that can be any real number.
+      * ``false`` - probabilities in *inputs* are expected to be non-negative, finite and have a non-zero-sum.
+  * **Type**: `bool`
+  * **Required**: *Yes*
+
+* ``global_seed``
+
+  * **Description**: global seed value.
+  * **Range of values**: non-negative integers
+  * **Type**: `unsigned int 64-bit`
+  * **Default value**: 0
+  * **Required**: *Yes*
+
+* ``op_seed``
+
+  * **Description**: operational seed value.
+  * **Range of values**: non-negative integers
+  * **Type**: `unsigned int 64-bit`
+  * **Default value**: 0
+  * **Required**: *Yes*
+
+**Inputs**:
+
+*   **1**: ``input`` - 1D or 2D tensor of type `T_IN` and shape `[class_size]` or `[batch_size, class_size]` with probabilities. Allowed values depend on the *log_probs* attribute. The values are internally normalized to have values in the range of `[0, 1]` with the sum of all probabilities in the given batch equal to 1. **Required.**
```

Comment:
Is there any more specific name suitable to be assigned as the "input" name like "probs"?
It's a detail, but input with name "input" when there are actually two inputs is a little bit misleading;)

Reply:
That's true, `probs` is a great idea!